docs/ref/contrib/admin/index.txt

```diff
@@ -906,13 +910,17 @@ subclass::
 
                   def lookups(self, request, model_admin):
                       if request.user.is_superuser:
-                          return super(AuthDecadeBornListFilter,
-                              self).lookups(request, model_admin)
+                          return super(
```

Comment:
This looks awkward. Maybe just leave it as is and when we drop Python 2 compatibility and switch to `super().` we can improve.